Gruntfile.js

```diff
@@ -34,13 +34,10 @@ module.exports = function(grunt) {
     browserify: {
       dist: {
         files: {
-          'tmp/videojs-dash.js': ['src/js/videojs-dash.js']
+          'tmp/videojs-dash.js': ['dist/es5/videojs-dash.js']
```

Comment:
In other `videojs-contrib-*` projects, `es5` is a top-level directory with `dist` containing the final browserified file. We may want to follow that convention everywhere to make moving from one to the other less confusing.